.github/workflows/docker-test.yml

```diff
@@ -30,9 +30,12 @@ jobs:
       - uses: actions/checkout@v4
         with:
           path: 'opensearch-benchmark-git'
-      - name: Docker Build and Test on ${{ matrix.architecture }}
+      - name: Docker Build ${{ matrix.platform }}
         run: |
             docker buildx version
             cp -a opensearch-benchmark-git/* ./
             echo "Disable VERSION arg to enter docker build test mode"
-            docker buildx build --platform ${{ matrix.architecture }} --build-arg BUILD_DATE=`date -u +%Y-%m-%dT%H:%M:%SZ` -f "docker/Dockerfile" .
+            PLATFORM=${{ matrix.platform }}
+            PLATFORM=`echo $PLATFORM | tr '/' '-'`
+            docker buildx build --platform ${{ matrix.platform }} --build-arg BUILD_ENV=testing --build-arg BUILD_DATE=`date -u +%Y-%m-%dT%H:%M:%SZ` -f "docker/Dockerfile" -t "osb/osb-$PLATFORM" -o type=docker .
+            docker images | grep "osb/osb-$PLATFORM"
```

Comment:
Probably can be streamlined, but fine for testing.
What does the `docker images` line do?  Is it simply to print the output of the `grep`?

Reply:
Yes @gkamat 

It is to setup the next step of running docker container based on the image just created.

In multi-arch build most of the time image has to be pushed upon completion to registry.
I force the image to be saved locally so we can use the image to run containers.

Once the container is up we can use it to run it.
This is a pre-req before that happened.

And I agree the code can be streamlined a bit, which I will improve in the next version.

Thanks.